inc/sitemaps/class-post-type-sitemap-provider.php

```diff
@@ -248,15 +248,13 @@ public function get_sitemap_links( $type, $max_entries, $current_page ) {
 				continue;
 			}
 
-			$posts_to_exclude = explode( ',', $options['excluded-posts'] );
-
 			foreach ( $posts as $post ) {
 
 				if ( WPSEO_Meta::get_value( 'meta-robots-noindex', $post->ID ) === '1' ) {
 					continue;
 				}
 
-				if ( in_array( $post->ID, $posts_to_exclude ) ) {
+				if ( in_array( $post->ID, $posts_to_exclude, true ) ) {
```

Comment:
If we are touching this line, please move it above the previous conditional, as this is much easier to check than doing a meta value check.

Reply:
Agree!